### DISCUSSION TITLE
Octoprint-Telegram plugin integration

### DISCUSSION THREAD

**Discussion by ismailk1** [mikulash/Octoprint-detector2#17] [Ideas]
https://github.com/fabianonline/OctoPrint-Telegram

This plugin is an alternative for Octoprint anywhere for me, it runs a telegram bot on the pi and sends messages, images alerts etc to your smartphone, and allows you to send commands, ask for status, images etc. back.

Would it be possible to integrate this plugin with the Telegram plugin so we don't have to use the email servers?

**Comment by mikulash** [comment]
Hi, in the next release my plugin will emit an event, which other plugins such as the one you mentioned can subscribe to. I could contact the author of Octoprint-telegram, whether he wants to notify users about my detection, but the source code is available here on github, so anyone can see it and connect his plugin to this detection without my knowledge or approval.
